post-process-pe.c

```diff
@@ -234,11 +234,11 @@ load_pe(const char *const file, void *const data, const size_t datasize,
 		        sizeof(EFI_IMAGE_DATA_DIRECTORY), &sz1))
 			debug(ERROR,
 			      "ctx->NumberOfRvaAndSizes (%zu) * sizeof(EFI_IMAGE_DATA_DIRECTORY) overflows\n",
-			      ctx->NumberOfRvaAndSizes);
+			      (size_t)ctx->NumberOfRvaAndSizes);
```

Comment:
NumberOfRvaAndSizes is always a UINT64 - so would this be better as unsigned long in all three cases (as you have it above)?

Reply:
yup